src/settings/JobProfiles/ViewJobProfile.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { Pane } from '@folio/stripes/components';
+
+// TODO: view component will be developed in UIDATIMP-133
+export default class ViewJobProfile extends Component {
```

Comment:
Lets avoid default exports
```suggestion
export class ViewJobProfile extends Component {
```